src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Crossgen2.sfxproj

```diff
@@ -37,6 +37,10 @@
           DestinationFolder="$(MicrosoftNetCoreAppRuntimePackNativeDir)"
           SkipUnchangedFiles="true" />
 
+    <!-- Run strip on the singlefilehost, in case it was built with symbols -->
+    <Exec Condition="'$(KeepNativeSymbols)' == 'true' and '$(HostOS)' != 'windows'"
+          Command="strip $(SingleFileHostSourcePath)" />
```

Comment:
This differs how we usually do strip for native bits. 

- Linux usually we preserve symbols in a separate file, we strip debug and unnecessary symbols, and we add the gnu debuglink. 
- macOS we prefer to use dsymutil.

At this point, if they asked to keep symbols, why are we trying to strip them like this? It seems like we have a problem, and this is just a workaround.

Reply:
Yeah, you're right, this won't work for Mac. And the logic is just too complicated to replicate for a workaround. Alright I'll close this out.